src/languages/java.js

```diff
@@ -72,7 +72,10 @@ export default function(hljs) {
     'requires',
     'exports',
     'do',
```

Comment:
export

Reply:
if you mean line 73, than no 😉. JPMS uses ´exports´ to define a package as available from other modules. See [JSL 7.7. Module Declarations](https://docs.oracle.com/javase/specs/jls/se19/html/jls-7.html#jls-7.7) and [JSL 3.9. Keywords](https://docs.oracle.com/javase/specs/jls/se19/html/jls-3.html#jls-ContextualKeyword)